src/pages/VaccinePage.js

```diff
@@ -14,25 +14,24 @@ import {
   DownloadIcon,
   TemperatureIcon,
   IconButton,
-  LightbulbIcon,
-  WifiIcon,
   CogIcon,
   Circle,
   FlexRow,
   AlarmClockIcon,
+  WifiIcon,
```

Comment:
Just added the use of `BlinkSensorButton` (Name is trademarked on this one, sorry) in this component - don't have to handle disabling/spinner etc again